flink-table-store-core/src/main/java/org/apache/flink/table/store/file/operation/FileStoreRead.java

```diff
@@ -33,6 +31,5 @@
 public interface FileStoreRead<T> {
 
     /** Create a {@link RecordReader} from partition and bucket and files. */
```

Comment:
Nit: modify the comment since the input argument has changed.
```suggestion
    /** Create a {@link RecordReader} from split. */
```